libcxx/docs/Status/Cxx23.rst

```diff
@@ -46,6 +46,7 @@ Paper Status
       clang doesn't issue a diagnostic for deprecated using template declarations.
    .. [#note-P2520R0] P2520R0: Libc++ implemented this paper as a DR in C++20 as well.
    .. [#note-P2711R1] P2711R1: ``join_with_view`` hasn't been done yet since this type isn't implemented yet.
+   .. [#note-P2770R0] P2770R0: Same as the above.
```

Comment:
I don't understand this note. I'm not sure the note will make sense once this is rendered as HTML?